File: app/src/main/java/com/agriminds/data/entity/CropDisease.java
package com.agriminds.data.entity;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "crop_diseases")
public class CropDisease {
    @PrimaryKey(autoGenerate = true)
    public int id;

    public String diseaseName;
    public String symptoms;
    public String treatment;
    public String prevention;
    public String imageUrl; // asset path or url
}

File: app/src/main/java/com/agriminds/data/entity/CropDisease.kt
package com.agriminds.data.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "crop_diseases")
data class CropDisease(
    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,
    val name: String,
    val cropType: String,
    val symptoms: String, // Comma-separated list
    val treatment: String,
    val prevention: String,
    val severity: String, // Low, Medium, High
    val imageUrl: String? = null
)
